core/src/apps/cardano/auxiliary_data.py

```diff
@@ -99,27 +93,22 @@ def _validate_catalyst_registration_parameters(
 async def show_auxiliary_data(
     ctx: wire.Context,
     keychain: seed.Keychain,
-    auxiliary_data: CardanoTxAuxiliaryDataType | None,
+    auxiliary_data_hash: bytes,
+    catalyst_registration_parameters: Optional[
```

Comment:
It's the exact same type, isn't it?
Would you prefer it because that would fit in a single line? Otherwise I think Optional[T] is more explicit than Union[T,None] (even with the `None` being implicit :rofl: )